app/styles/utils.scss

```diff
@@ -80,18 +94,49 @@ $units: 0, 1, 2, 3, 4;
 
 .font-bold { font-weight: bold; }
 .font-mono { font-family: var(--font-monospace); }
+.leading-none { line-height: 1; }
 .no-underline { text-decoration: none; }
+.underline { text-decoration: underline; }
 .text-left { text-align: left; }
 .text-center { text-align: center; }
 .text-right { text-align: right; }
 .whitespace-no-wrap { white-space: nowrap; }
 
 .bg-transparent { background-color: var(--transparent); }
-.bg-base00 { background-color: var(--base00); }
-.bg-base01 { background-color: var(--base01); }
-.bg-base02 { background-color: var(--base02); }
+
+.fill-current { fill: currentColor; }
 
 .cursor-default { cursor: default; }
 .cursor-pointer { cursor: pointer; }
+.cursor-col-resize { cursor: col-resize; }
 
 .z-10 { z-index: 10; }
+
+.appearance-none {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+.pointer-events-none { pointer-events: none; }
+
+.outline-none { outline: 0; }
+
+/*
+  Utilities not part of Tailwind:
+*/
+
+.text-inherit { color: var(--inherit); }
+
+.bg-base00 { background-color: var(--base00); }
```

Comment:
Tailwind supports setting your own colors to be used for both background-color and text color, so these could actually be part of tailwind.